src/display/api.js

```diff
@@ -2812,12 +2829,6 @@ class WorkerTransport {
     });
   }
 
-  getPageXfa(pageIndex) {
-    return this.messageHandler.sendWithPromise("GetPageXfa", {
```

Comment:
Also, if you're removing sending of this message the following handler looks like dead code: https://github.com/mozilla/pdf.js/blob/3538ef017fa05787d2e0c2b6bdf2d904753da489/src/core/worker.js#L504-L508